src/main/java/org/veupathdb/service/eda/ds/plugin/standalonemap/markers/CategoricalRatioAggregator.java

```diff
@@ -0,0 +1,40 @@
+package org.veupathdb.service.eda.ds.plugin.standalonemap.markers;
+
+import java.util.Set;
+
+/**
+ * Map marker aggregator taking categorical variable and providing a ratio of number of elements with values in a
+ * set of numerator values over number of elements with values in a set of denominator values.
+ */
+public class CategoricalRatioAggregator implements MarkerAggregator<Double> {
```

Comment:
frustratingly, id like to name this something other than ratio.. ideally proportions maybe. but we also have proportions in another context w a slightly different semantic intent..

Reply:
maybe we can figure how to collapse these two categorical aggregators into one?

Reply:
or reorganize them somehow? im not sure what i want yet, sorry..

Reply:
I did some re-naming. Let me know if you have thoughts about where we ended up. I made them separate to try to reduce duplication with the different map marker plugins.